src/mono/mono/component/hot_reload.c

```diff
@@ -529,14 +529,14 @@ hot_reload_close_all (MonoImage *base_image)
 	if (!info)
 		return;
 	for (GList *ptr = info->delta_info; ptr; ptr = ptr->next) {
-		DeltaInfo *info = (DeltaInfo *)ptr->data;
-		if (!info)
+		DeltaInfo *delta_info = (DeltaInfo *)ptr->data;
```

Comment:
What was the problem here ?

Reply:
That change was mainly aligning the last DeltaInfo *info to use same naming as all others in this file, the other change in this file fix a local hiding issue,  warning C4456: declaration of 'info' hides previous local declaration, that was very common in the source and fixed in this PR.